declarations/Parler.history.json

```
@@ -0,0 +1,14 @@
+{
+    "Privacy Policy": {
+      "fetch": "https://parler.com/documents/privacypolicy.pdf",
+      "validUntil": "2023-12-12T18:32:42Z"
+    },
+    "Terms of Service": {
+      "fetch": "https://parler.com/documents/termsofservice.pdf",
+      "validUntil": "2023-12-12T18:32:42Z"
+    },
+    "Community Guidelines": {
+      "fetch": "https://parler.com/documents/guidelines.pdf",
+      "validUntil": "2023-12-12T18:32:42Z"
+    }
+}
```

Hi @smmkolesnikov, all checks have failed because the history file does not have the expected structure. 
Each term type should be declared within an array, rather than an object:
```suggestion
{
  "Privacy Policy": [
    {
      "fetch": "https://parler.com/documents/privacypolicy.pdf",
      "validUntil": "2023-12-12T18:32:42Z"
    }
  ],
  "Terms of Service": [
    {
      "fetch": "https://parler.com/documents/termsofservice.pdf",
      "validUntil": "2023-12-12T18:32:42Z"
    }
  ],
  "Community Guidelines": [
    {
      "fetch": "https://parler.com/documents/guidelines.pdf",
      "validUntil": "2023-12-12T18:32:42Z"
    }
  ]
}
```

---

Hi Matti and Nicolas!

@MattiSG hmm that is a good point. Last time we discussed it with colleagues there was only a blog, but now it seems they are going to relaunch the social network too, after all. @pg-adrian @DariaDergacheva please take a look at this, shall we then keep tracking Parler?

@Ndpnt thank you! I believe there was the same problem with terminating Twitter [here](https://github.com/OpenTermsArchive/pga-declarations/pull/236/commits/f3a6171d046f880ba45e29897b3612a8548367eb), right?

---

Exactly!

---

Hi thank you @MattiSG for the pointer, that had escaped us that Parler is now also planning to come back as a social media site. So we would suggest to not terminate Parler then and just check regularly for when they publish their policies.